assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.xml

```diff
@@ -204,67 +300,265 @@
+            <param name="investigationResult">Result of the investigation amongst the different properties which contains found key and token.</param>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ProcessChildrenTokenProperties(Newtonsoft.Json.Linq.JObject,Gnmi.Path,System.Int32,System.Collections.Generic.List{Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Utils.PathKeyPart},System.Collections.Generic.List{Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.KeyAndToken})">
+            <summary>
+            Process the properties of a JSON object.
+            </summary>
+            <param name="objectToken">JSON object to be processed.</param>
+            <param name="entityPath">Path filter in the data grid entity that specifies what JSON token value should be returned for a specific level.</param>
+            <param name="pathLevel">Location of the JSON object in the path filter.</param>
+            <param name="pathKeyParts">Parts of the primary key.</param>
+            <param name="keysAndTokens">Found JSON token values with their key parts.</param>
+            <returns>Boolean indicating if token values were found in the requested path.</returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.RemoveModulePrefix(System.String)">
+            <summary>
+            Removes the module name from a JSON property name.
+            </summary>
+            <param name="name">Name of the JSON property.</param>
+            <returns>Name of the JSON property without module name.</returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ValidateMatchingKey(System.String,System.String,Gnmi.PathElem)">
+            <summary>
+            Verifies if the current property matches with the path element.
+            </summary>
+            <param name="propertyName">Name of the property to be verified.</param>
+            <param name="propertyValue">Value of the property to be verified.</param>
+            <param name="pathElem">Path element of the filter.</param>
+            <returns>Boolean indicating if the property matches with the path element.</returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.AddKeyFromToken(Newtonsoft.Json.Linq.JObject,System.Collections.Generic.List{Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Utils.PathKeyPart},System.Int32)">
+            <summary>
+            Looks in the properties of a token. When it contains exactly one property that is a string or int, a key will be created based on that, or the index of the row will be used.
```

Comment:
Do you mean that if there is no such property, the index of the row will be used?